tests/client/test_delphi_epidata.py

```diff
@@ -2,10 +2,15 @@
 
 # standard library
 import unittest
+import time
```

Comment:
what are all these changes for?

Reply:
1. I realized that cache_timeout can take `int`, `timedelta`, and `datetime`, so I wanted to be clear about that.
2. Removed unused imports.

Reply:
Oh this is a different file, oops. Um, I wrote my own benchmarking tests in this file and then reverted it, but it looks like some imports stayed. Thanks for catching.